src/privatesend-client.cpp

```diff
@@ -328,24 +328,24 @@ void CPrivateSendClient::CheckPool()
 //
 void CPrivateSendClient::CheckTimeout()
 {
+    if(fMasternodeMode) return;
+
     CheckQueue();
 
-    if(!fEnablePrivateSend && !fMasternodeMode) return;
+    if(!fEnablePrivateSend) return;
```

Comment:
If `CheckQueue` is intensive (or maybe even not), would it be better to move this above that call?

Reply:
Not very intensive (Queue TTL is low and there are not so many of them) but anyway, that's intentional. User can stop PrivateSend at any moment and we still have to check the queue vector and drop expired ones.